modules/appeals_api/config/routes.rb

```diff
@@ -119,7 +119,8 @@
         resources '10182', only: %i[create show], controller: controller_path do
           collection do
             post 'validate'
```

Comment:
I went ahead and disabled all of the rest of these here, since we won't make them public until after launch

Reply:
We may have to bump to a new minor version in that case, according to this API standard:
> Backward-compatible changes that introduce new endpoints or new fields within existing endpoints must be released within a new minor version.

Reply:
I talked with Drew a bit yesterday about `download`, and it sounds like this may change soon - will revisit once I hear the final word - I _do_ think these will be re-enabled here or ported back to decision reviews v2